finish/src/test/java/it/io/openliberty/guides/inventory/SystemResourceIT.java

```diff
@@ -14,13 +14,9 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import java.net.Socket;
-import java.security.KeyStore;
 import java.security.KeyStoreException;
```

Comment:
Remove if not using the SSL setup or keystore.

```suggestion
```

Reply:
removed